lib/bundle/commands/add.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Bundle
+  module Commands
+    module Add
+      module_function
+
+      def run(*args, global: false, file: nil)
+        raise UsageError, "No arguments were specified!" if args.blank?
+
+        type = :brew # default to brew
+        name = args.first # only support one formula at a time for now
+        options = {} # we don't currently support passing options
```

Comment:
Should we handle different types of formulae? Is there an existing pattern for how you'd like this handled?

Reply:
I think it's fine to just support `brew` for now and people can extend it later if desired. As long as it's documented it can `raise UsageError` which will print the help text.